examples/plugin/wasi-logging/Cargo.toml

```diff
@@ -4,4 +4,4 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-wit-bindgen = { git = "https://github.com/bytecodealliance/wit-bindgen",  rev="2ec8e3e256f493b62c561a68300ec09c03d040c4" }
```

Comment:
You should pick an revision due to the wit-bindgen usually has some breaking changes between versions.